src/python/grpcio/commands.py

```diff
@@ -74,46 +77,52 @@ class CommandError(Exception):
 
 # TODO(atash): Remove this once PyPI has better Linux bdist support. See
 # https://bitbucket.org/pypa/pypi/issues/120/binary-wheels-for-linux-are-not-supported
-def _get_grpc_custom_bdist_egg(decorated_basename, target_egg_basename):
-  """Returns a string path to a .egg file for Linux to install.
+def _get_grpc_custom_bdist(decorated_basename, target_bdist_basename):
+  """Returns a string path to a bdist file for Linux to install.
 
-  If we can retrieve a pre-compiled egg from online, uses it. Else, emits a
+  If we can retrieve a pre-compiled bdist from online, uses it. Else, emits a
   warning and builds from source.
   """
+  # TODO(atash): somehow the name that's returned from `wheel` is different
```

Comment:
I don't understand this. Is something wrong with `wheel`? If so, link to a wheel bug report? Is `wheel` missing something that you feel we need? If so, link to a wheel feature request? If neither, I'm really unclear on what work is to be done here that we need TODO but are not able to do at this time?